assets/js/blocks/mini-cart/quantity-badge/index.tsx

```diff
@@ -8,20 +8,20 @@ import { Icon } from '@wordpress/icons';
  * Internal dependencies
  */
 import './style.scss';
-import { IconType } from '.././types';
+import { IconType, ColorItem } from '.././types';
 
 interface Props {
 	count: number;
 	icon?: IconType;
-	iconColor?: string;
-	productCountColor?: string;
+	iconColor: ColorItem | { color: undefined };
+	productCountColor: ColorItem | { color: undefined };
 }
 
 const QuantityBadge = ( {
 	count,
 	icon,
-	iconColor,
-	productCountColor,
+	iconColor = { color: undefined },
+	productCountColor = { color: undefined },
```

Comment:
Again, consistency of types :). Also will need to make sure if importing types that they are exposed in a frontend build friendly location.